### DISCUSSION TITLE
How to Create Collapsable Hamburger Nav Bar

### DISCUSSION THREAD

**Discussion by numenor-dev** [twilio-labs/paste#2959] [Q&A] [answered]
Hi,

I am actually trying to create the same style of collapsable navigation that is on the Twilio Paste site itself https://paste.twilio.design/

When going below a width of roughly 1024px, the navigation bar at the top of the page collapses into a Hamburger style button that can be toggled to open the navigation links.

I'm not seeing how to recreate this in the Twilio Paste docs.

**Comment by loreina** [comment, marked as answer]
Hi @numenor-dev,

It’s not a component in our system, but you can see how we put it together using Paste directly in our [codebase](https://github.com/twilio-labs/paste/blob/main/packages/paste-website/src/components/site-wrapper/site-header/HamburgerToggle.tsx).

**Reply by numenor-dev** [reply]
Hi @loreina,

Thank you! This is great. One quick follow up question, is there a way to just auto-collapse a section based on specific breakpoints? The section I'm wanting to modify on our site currently has a manual button toggle to collapse the section but ideally it would be great for it to auto-collapse on smaller viewports.
